.github/workflows/upgrade_version.yaml

```diff
@@ -24,7 +24,6 @@ jobs:
 
                   # Update the version in CmakeLists.txt
                   sed -i 's/ VERSION [0-9]*\.[0-9]*\.[0-9]*$/ VERSION ${{ github.event.inputs.new_version }}/' api/cpp/CMakeLists.txt
-                  sed -i 's/SLINT_VERSION [0-9]*\.[0-9]*\.[0-9]*$/SLINT_VERSION ${{ github.event.inputs.new_version }}/' api/cpp/CMakeLists.txt
```

Comment:
The $ doesn't match with the project version.

Reply:
Ooops